### ISSUE TITLE
Stickers?

### ISSUE THREAD

**Issue by Riz-waan** [fireship-io/git-sticker#865]
Are you guys still giving out stickers, I haven't gotten anything could you check if I qualify? Thought I did. Thanks!

**Comment by juwul** [comment]
Yes, me too. I did PR back in the start of March.

**Comment by amturner** [comment]
I haven't received any stickers here either.

**Comment by nivagator** [comment]
I didn't receive a sticker either.

**Comment by OfficialCRUGG** [comment]
Same here...

**Comment by ariskycode** [comment]
Any news on this?

**Comment by juwul** [comment]
Yeah, its a scam :D

**Comment by OfficialCRUGG** [comment]
Yup it appears to be exactly that. We haven't got any form of Communication on this in ages.

**Comment by OfficialCRUGG** [comment]
I also just noticed that it's been half a year.

**Comment by amturner** [comment]
It honestly is a shame considering they got multiple people to participate. Here I am... still with no sticker...

**Comment by OfficialCRUGG** [comment]
What impresses me the most is that it seems that there is an actual Company doing this. I don't know much about law outside of Germany, but here it is definitely illegal to promise to give away things as a company and then just scam the people. Even if it's just Stickers.

**Comment by Riz-waan** [comment]
Well, they have a slack channel if you guys wanna ask on there?

https://join.slack.com/t/angularfirebase/shared_invite/zt-ezk5n9dd-pFWJ~593VdbnTM2bw3c7xA

**Comment by OfficialCRUGG** [comment]
That's actually what I just did. I'm still waiting for a reply. The Slack appears to be very inactive though.

**Comment by juwul** [comment]
Post slack server name

**Comment by Riz-waan** [comment]
@juwul check out the invite link i posted.

**Comment by niekvangogh** [comment]
Scam to get viewers address, pog

**Comment by amturner** [comment]
I guess whatever message that was sent was ignored... 😐

**Comment by StavenX** [comment]
They did give out stickers. I submitted a pull request not long after the video was out, so I was one of the first ones. 
You might just have gotten unlucky, but this was in no way a scam.

That's what happens with giveaways though. Sometimes we win, but mostly we lose.

**Comment by OfficialCRUGG** [comment]
> They did give out stickers. I submitted a pull request not long after the video was out, so I was one of the first ones.
> You might just have gotten unlucky, but this was in no way a scam.
> 
> That's what happens with giveaways though. Sometimes we win, but mostly we lose.

Yes I know there was one "wave". But I was also very early and it's not like just a few people get Stickers. The holographic ones are limited. Yes. But not the others. And it was promised, that if this is done, the Repo would be deleted, which it still isn't. And after trying to contact the Person responsible for this via multiple ways (Slack, GitHub) they always seem to ignore requests related to this Repository. They send messages in Slack, but don't respond to the Messages I send at the same time.

**Comment by Riz-waan** [comment]
> They did give out stickers. I submitted a pull request not long after the video was out, so I was one of the first ones.
> You might just have gotten unlucky, but this was in no way a scam.
> 
> That's what happens with giveaways though. Sometimes we win, but mostly we lose.

Yea that makes sense, however, I calculated my position, and although I didn't make it within the holographic sticker, I still made it within the reorder. So, I think it was like the first 250 and then another 500 regular stickers, and I think I was like within 500 so nowhere close to the limit of 750.

Also, I think they shouldn't have merged the pull request if they ran out of stickers. A better option would have been to just close it with a comment using a script. Merging it makes it look like they accepted you in the wave to get a sticker.

**Comment by Riz-waan** [comment]
Like how currently anyone opening a pull request is automatically closed.

**Comment by amturner** [comment]
I had a pull request that was merged in. 
![image](https://user-images.githubusercontent.com/34004239/95465482-1c8fb080-0973-11eb-8239-9c4bb6516261.png)

**Comment by Riz-waan** [comment]
@StavenX #86                                                         

@amturner #204
@nivagator #276
@Riz-waan #455 
@OfficialCRUGG #463 
@niekvangogh #497 


For everyone who participated in this thread, the above people's pull requests were merged, which I thought would mean accepted. Has anyone other than @StavenX  gotten stickers? Okay, the description said first 200, so it makes sense @StavenX  got one, but it also said others would get a sticker until they ran out. I'm sure the video specified a number greater than 3. Atleast @amturner should have gotten one of the normal nonholographic stickers. I'm sure we can give them the benefit of the doubt, but communication would be nice.

**Comment by OfficialCRUGG** [comment]
To be honest, I don't even want the Stickers anymore. I just what to know what's up with this Repository. And the person responsible seems to be purposefully ignoring this. If you don't want to send out the Stickers anymore, just delete this Repository and let it be done. Or at least tell us what's up.

**Comment by OfficialCRUGG** [comment]
Also, let me quote @codediodeio in #123:
> I will delete the repo after the stickers go out next week. 💣
(That was on March 21st.)

**Comment by Janbuller** [comment]
I'm just replying, to say, I got my sticker.

**Comment by amturner** [comment]
@Janbuller So you received your sticker? If that's the case... I wonder if there was anything wrong with our pull requests...?

**Comment by ariskycode** [comment]
@Janbuller When did you get yours?

The last merge was March 29th

**Comment by amturner** [comment]
@ariskycode Their pull request, #199, was created on March 17th and merged on March 22nd. My pull request, #204, was created on March 17th and merged on March 28th.
